monday/query_joins.py

```diff
@@ -443,3 +443,15 @@ def delete_group_query(board_id, group_id):
         }
     }''' % (board_id, group_id)
     return query
+
+
+def get_complexity_query():
+    query = '''query
```

Comment:
Could you put the second "query" on the line below so that it matches the formatting of other queries in this file?

Reply:
done